packages/mgt-components/src/components/mgt-people/mgt-people.tests.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import { fixture, html, expect, oneEvent } from '@open-wc/testing';
+import { MockProvider, Providers } from '@microsoft/mgt-element';
+import { MgtPeople, registerMgtPeopleComponent } from './mgt-people';
+
+describe('mgt-people - tests', () => {
+  registerMgtPeopleComponent();
+  Providers.globalProvider = new MockProvider(true);
+
+  it('should render with overflow', async () => {
+    const mgtPeople = await fixture(html`<mgt-people
+          people-queries="Lidia, Megan, Lynne, Brian, Joni">
+        </mgt-people>
+        `);
+    // @ts-expect-error TS2554 expects 3 arguments got 2 https://github.com/open-wc/open-wc/issues/2746
```

Comment:
Bump `@open-wc/testing-helpers` to `3.0.1`. This is fixed.

Reply:
After bumping, this still fails. I'll revert, will update in a separate PR